content/docs/get-started/aws/review-project.md

```diff
@@ -82,16 +82,31 @@ class Program
 This Pulumi program creates an S3 bucket and exports the name of the bucket.
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
```

Comment:
Can we put this on the install python page of the getting started instead?  We may as well just tell folks to install this upfront. 

We should also make sure this requirement is clearly called out on the Python language page.

Reply:
I've updated the Python language page and removed it from the getting started pages